src/Panel/panel.scss

```diff
@@ -107,11 +107,7 @@ $content-title-color: #313D4F !default;
 
     .panel-header-title {
       font-size: 16px;
-      font-weight: 700;
-    }
-
-    .panel-header-content {
-      margin-left: 5px;
```

Comment:
ok, then I'll merge this!
@veej can you open on AlinIQ and IperCron an issue to fix the style of the components you've touched (please list them all in the issue and point to this PR so we don't forget the changes)